tls_socket.js

```diff
@@ -221,12 +217,12 @@ function _getSecureContext (options) {
     }
     if (!options.sessionIdContext) {
        	options.sessionIdContext = 'haraka';
-    };
+    }
     if (!options.sessionTimeout) {
        	// options.sessionTimeout = 1;
-    };
+    }
 
-    secureContext = tls.createSecureContext(options);
+    var secureContext = tls.createSecureContext(options);
     return secureContext;
```

Comment:
Might as well remove the variable declaration entirely and just `return tls.createSecureContext(options);`